src/app/backend/integration/manager.go

```diff
@@ -0,0 +1,70 @@
+package integration
+
+import (
+	"fmt"
+
+	"github.com/kubernetes/dashboard/src/app/backend/client"
+	"github.com/kubernetes/dashboard/src/app/backend/integration/api"
+	"github.com/kubernetes/dashboard/src/app/backend/integration/metric"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// IntegrationManager is responsible for management of all integrated applications.
+type IntegrationManager interface {
+	// List returns list of all supported integrations.
+	List() []api.Integration
+	// GetState returns state of integration based on its' id.
+	GetState(id api.IntegrationID) (*api.IntegrationState, error)
+	// Metric returns metric manager that is responsible for management of metric integrations.
+	Metric() metric.MetricManager
+}
+
+// Implements IntegrationManager interface
+type integrationManager struct {
+	metric metric.MetricManager
+}
+
+// List implements integration manager interface. See IntegrationManager for more information.
+func (self *integrationManager) List() []api.Integration {
+	result := make([]api.Integration, 0)
+
+	// Append all types of integrations
+	result = append(result, self.Metric().List()...)
```

Comment:
Yeah. I meant that we could have integrations-list.go file with this array. Filled the same way you do here, but in a separate file so that the place I register integrations is only one.